pom.xml

```diff
@@ -94,11 +94,25 @@
     </dependency>
 
     <dependency>
-      <groupId>org.eclipse.jetty</groupId>
-      <artifactId>jetty-client</artifactId>
-      <version>${jetty.version}</version>
+      <groupId>com.squareup.okhttp3</groupId>
+      <artifactId>okhttp</artifactId>
+      <version>3.9.0</version>
```

Comment:
3.8.1 is used at Presto 0.187 but guess it's compatible

Reply:
Yeah. Presto doesn't use OkHttp for server side code, so it might be OK.